app/views/search/search_fast.html.erb

```diff
@@ -0,0 +1,148 @@
+<% title "Search" %>
+
+<p>Pages: <%= @results.total_pages %></p>
+<p>CurPage: <%= @results.current_page %></p>
+
+<% @logs.each do |log| %>
+<p><%= log %></p>
```

Comment:
We should remove development stuff now it's in production, or wrap it in a `if Rails.env.development?`. This and CurPage can both go.